client/influxdb.go

```diff
@@ -112,13 +99,21 @@ func (c *Client) Write(writes ...Write) (*Results, error) {
 	defer resp.Body.Close()
 
 	var results Results
-	dec := json.NewDecoder(resp.Body)
-	dec.UseNumber()
-	err = dec.Decode(&results)
+	body, _ := ioutil.ReadAll(resp.Body)
+	if len(body) > 0 {
```

Comment:
Should also add that if there IS an error, that is when we write out a body, and then you need to decode it.